spellcheck/spellcheck_types.h

```diff
@@ -8,3 +8,11 @@
 
 using MisspelledWord = std::pair<int, int>;
 using MisspelledWords = std::vector<MisspelledWord>;
+
+namespace Spellchecker {
+
+#if QT_VERSION < QT_VERSION_CHECK(5, 10, 0)
+using QStringView = const QString&;
+#endif
+
+} // namespace Spellchecker
```

Comment:
@mymedia2 
I think there will be a compile error here for Windows.
If you want to use namespaces, you should probably duplicate it for `Platform::Spellchecker` or just remove namespace.

```
#if QT_VERSION < QT_VERSION_CHECK(5, 10, 0)
namespace Spellchecker {
using QStringView = const QString&;
} // namespace Spellchecker

namespace Platform::Spellchecker {
using QStringView = const QString&;
} // namespace Platform::Spellchecker
#endif
```

Reply:
:sweat: okay, it seems I've to remove the namespace definition. I put the type alias into the Spellchecker namespace to avoid an effect on large code base.